ci/builders/linux_web_engine.json

```diff
@@ -41,28 +41,50 @@
                 ],
                 "tasks": [
                     {
-                        "name": "compile web_tests",
+                        "name": "check licenses",
                         "parameters": [
-                            "run",
-                            "compile_tests"
+                            "check-licenses"
                         ],
                         "scripts": [
                             "flutter/lib/web_ui/dev/felt"
                         ]
                     },
                     {
-                        "name": "check licenses",
+                        "name": "web engine analysis",
                         "parameters": [
-                            "check-licenses"
+                            "analyze"
                         ],
                         "scripts": [
                             "flutter/lib/web_ui/dev/felt"
                         ]
-                    },
+                    }
+                ]
+            },
+            "tests": []
+        },
+        {
+            "archives": [],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "gclient_variables": {
+                "download_android_deps": false
+            },
+            "gn": [],
+            "name": "web_tests",
+            "ninja": {},
+            "generators": {
+                "pub_dirs": [
```

Comment:
I don't think we need these. `felt` will automatically run pub on anything it needs

Reply:
ack